api/v1beta1/slicegateway_types.go

```diff
@@ -53,6 +53,8 @@ type SliceGatewayConfig struct {
 	SliceGatewayNodePort int `json:"sliceGatewayNodePort,omitempty"`
 	// Remote Node IP
 	SliceGatewayRemoteNodeIP string `json:"sliceGatewayRemoteNodeIp,omitempty"`
+	// Remote Node IPs
+	SliceGatewayRemoteNodeIPs []string `json:"sliceGatewayRemoteNodeIps,omitempty"`
 	// Remote Node Port
```

Comment:
Don't we need multiple node ports?

How about we rename the field to `SliceGatewayRemoteAddresses` and include `ip:port` as the value?